src/mmw/apps/modeling/mapshed/calcs.py

```diff
@@ -254,11 +254,9 @@ def ls_factor(stream_length, area, avg_slope, m):
         return ls
 
 
-def stream_length(geom, drb=False):
```

Comment:
Was `LS` the only calc that had been using the switch?  I seem to recall another calculation that made the same evaluation on location.